### PR TITLE
Add preliminary documentation

### PR DESCRIPTION
**Pull request by NicolasT**
This PR lays the groundwork for in-tree documentation of this project, using [Sphinx](http://www.sphinx-doc.org).

### CODE REVIEW

**Comment by NicolasT** [comment]
Retest this please.

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/swift-tox/155/
Test PASSed.

**Comment by JordanP** [comment]
```
Running Sphinx v1.3b3
making output directory...
no previously-included directories found matching 'doc/_build/*'
warning: no previously-included files matching '*~' found anywhere in distribution
loading pickled environment... not yet created

Warning, treated as error:
WARNING: 'default' html theme has been renamed to 'classic'. Please change your html_theme setting either to the new 'alabaster' default theme, or to 'classic' to keep using the old default.

ERROR: InvocationError: '/opt/stack/ScalitySproxydSwift/.tox/doc/bin/sphinx-build -W -b html -d /opt/stack/ScalitySproxydSwift/.tox/doc/tmp/doctrees . /opt/stack/ScalitySproxydSwift/.tox/doc/tmp/html'
_____________________________________________________________ summary _____________________________________________________________
ERROR:   doc: commands failed
```
Any idea to work around this ?

**Comment by JordanP** [comment]
I had to remove the '-W' from the sphinx-build command

File: doc/installation.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+2. Modify your Swift :file:`object-server.conf` to use the new object server:
+
+   .. code-block:: ini
+
+       [app:object-server]
+       use = egg:swift_scality_backend#sproxyd_object
+
+3. Set the Sproxy host(s) and Sproxy path to connect to in the
+   ``[app:object-server]`` section in the same file. If your system supports it
```
Comment (JordanP) [inline doc/installation.rst]: We just repeat the file name, again. I'd replace `in the same file` with :file:`object-server.conf`

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/swift-tox/160/<h2>Build result: FAILURE</span></h2>[...truncated 21 lines...]Triggering swift-tox » py27-swift1.13.1,ubuntu14-largeTriggering swift-tox » py27-swift1.13.1,ubuntu12-largeTriggering swift-tox » pep8,ubuntu14-largeTriggering swift-tox » py27-swift2.2.0,centos7-largeTriggering swift-tox » py27-swifthead,ubuntu14-largeTriggering swift-tox » py27-swifthead,ubuntu12-largeTriggering swift-tox » py26-swift1.13.1,centos6-largeConfiguration swift-tox » py27-swifthead,centos7-large is still in the queue: All nodes of label ‘centos7-large’ are offlineConfiguration swift-tox » py27-swifthead,centos7-large is still in the queue: centos7-large-339 is offlineConfiguration swift-tox » py27-swifthead,centos7-large is still in the queue: All nodes of label ‘centos7-large’ are offlineswift-tox » py27-swifthead,centos7-large completed with result SUCCESSswift-tox » py27-swift1.13.1,centos7-large completed with result SUCCESSswift-tox » py26-swifthead,centos6-large completed with result SUCCESSswift-tox » py27-swift2.2.0,ubuntu14-large completed with result SUCCESSswift-tox » py27-swift2.2.0,ubuntu12-large completed with result SUCCESSswift-tox » py26-swift2.2.0,centos6-large completed with result SUCCESSswift-tox » py27-swift1.13.1,ubuntu14-large completed with result SUCCESSswift-tox » py27-swift1.13.1,ubuntu12-large completed with result FAILUREswift-tox » pep8,ubuntu14-large completed with result SUCCESSswift-tox » py27-swift2.2.0,centos7-large completed with result SUCCESSswift-tox » py27-swifthead,ubuntu14-large completed with result SUCCESSswift-tox » py27-swifthead,ubuntu12-large completed with result SUCCESSswift-tox » py26-swift1.13.1,centos6-large completed with result SUCCESSSetting status of 189aeb1193625753d1a7658617d9997f79836bec to FAILURE with url https://37.187.159.67:5443/job/swift-tox/160/ and message: Merged build finished.
Test FAILed.

**Comment by JordanP** [comment]
LGTM

**Pull request merged**